app-admin/lnav/lnav-0.11.1-r1.ebuild

```diff
@@ -11,7 +11,7 @@ SRC_URI="https://github.com/tstack/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}
 
 LICENSE="BSD-2"
 SLOT="0"
-KEYWORDS="amd64 x86"
+KEYWORDS="amd64 ~arm ~arm64 x86"
```

Comment:
Is it OK to add unstable keywords to an otherwise stable ebuild like this? Or should I make a `-r2` for this change? I'm happy to do the latter, I just wasn't sure what the right path was.

Reply:
If you've personally run the test suite on arm/arm64: yes, it's ok, you can add them

If not, just let the arch teams do it (don't add them yourself at all), and your only purpose in the bug is if you know lnav is totally broken on arm/arm64 and to scream loudly :)

Reply:
Thanks Sam! I don't have a Linux ARM host to do the test suite with. It sounds like what I should do is drop my second commit from this PR (I'd still like to do the first one, which just drops the older of the two stable versions) and let the BZ do the rest.

Reply:
Sounds good!

Reply:
Alright, I've force pushed and dropped the keyword commit, leaving just the one to drop the older stable version.

Thanks for the help!